### PR TITLE
DeepEval Evaluation Functionality [REBASED]

### PR DESCRIPTION
**Pull request by outlawhayden**
**Now Rebased for 1 Commit PR**
Hey All -

We found a super sweet library called [DeepEval](https://docs.confident-ai.com/) that could fill a lot of the need for a model evaluation framework. Not only will it look at response quality, but will also evaluate the retrieval of context vs the answer, and some pretty cutting edge math for cool metrics including bias, relevancy, toxicity, and more.

To run the current setup, navigate to the evaluate folder, and then run deepeval test run test_model_cached.py, where the last parameter is one of the python files that start with 'test' in that folder. One of them looks at more of the advanced metrics, one of them reads in a list of prompts from a .csv file into various tests, and one of them bootlegs the original CLI interface in getanswer/main.py to evaluate a query you input live against some metric (answer relevancy is hardcoded in there now but changeable).

Still definitely in the works, and we still have some wrinkles to smooth out, but we wanted to share our progress and create a space for brainstorming where to go in the future!

Underneath it really is just more external API calls to other models to do the more advanced evaluation. The good news is that it's using other OpenAI architecture, you just might have to key in your API key as an environment variable before calling the library (reading from the global env file is still not working, one of the aforementioned wrinkes), but other than that it should work out of the box.

### CODE REVIEW

File: packages/googlecloud/functions/getanswer/evaluate.py [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+            actual_output, retrieval_context = route_question(
+                voting_roll_df,
+                db_fc,
+                db_cj,
+                db_pdf,
+                db_pc,
+                db_news,
+                query,
+                RESPONSE_TYPE_DEPTH,
+                k=5,
```
Comment (aronwc) [inline packages/googlecloud/functions/getanswer/evaluate.py:89]: good for debugging, but probably drop so we use the default k.

File: packages/evaluate/test_example.py
Comment (outlawhayden) [inline packages/evaluate/test_example.py:1]: delete file

File: packages/googlecloud/functions/getanswer/test_model_baseline_tests.py
Comment (outlawhayden) [inline packages/googlecloud/functions/getanswer/test_model_baseline_tests.py:1]: delete file

File: packages/googlecloud/functions/getanswer/test_model_cache.py
Comment (outlawhayden) [inline packages/googlecloud/functions/getanswer/test_model_cache.py:1]: delete file

File: packages/googlecloud/functions/getanswer/test_model_live.py
Comment (outlawhayden) [inline packages/googlecloud/functions/getanswer/test_model_live.py:1]: delete file

**Pull request merged**